src/ManagerWindows.cc

```diff
@@ -331,23 +331,35 @@ RootWO::handleLeaveEvent(XCrossingEvent *ev)
  * Makes sure the Geometry is inside the screen.
  */
 void
-RootWO::placeInsideScreen(Geometry& gm, bool without_edge)
+RootWO::placeInsideScreen(Geometry& gm, bool without_edge, bool fullscreen, bool maximized_virt, bool maximized_horz)
 {
 	uint head_nr = X11::getNearestHead(gm.x, gm.y);
```

Comment:
Hm... I can't say (especially because i have never had 3 screens). But yeah it seems to make more sense to find nearest head based on the center of the window, instead of top left corner.